tokio-signal/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![doc(html_root_url = "https://docs.rs/tokio-signal/0.2.8")]
+#![deny(missing_debug_implementations)]
```

Comment:
minor nit: but this should be grouped with the deny below?

Reply:
Personally a fan on having lints one on each line, so I copied this without checking (I'm surprised `rustfmt` didn't fix this), but I'll update for consistency